target_chains/fuel/sdk/js/src/index.ts

```diff
@@ -0,0 +1,73 @@
+import PYTH_CONTRACT_ABI from "./abi/pyth-contract-abi.json";
+
+/** Address of the Pyth contract on Fuel Sepolia (testnet). */
+export const PYTH_CONTRACT_ADDRESS_SEPOLIA =
+  "0xdf202803b7a452a1ed2e09291731dbac5ceb13d036de3a9a31f74e6057c909af";
+
+/** Asset ID of ETH on Fuel. */
+export const FUEL_ETH_ASSET_ID =
+  "0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07";
+
+export { PYTH_CONTRACT_ABI };
+export * from "./types";
+// Doing this is not ideal, but it's the only way to get the types to work
+export {
```

Comment:
this is unfortunate but i tried doing `export * from ./types/PythAbiContract` and it doesn't work still, also couldn't get it to work with subpaths as for some reason the PythContractAbi.d.ts wasn't build when `tsc` is ran, happy to get any other suggestions to make this cleaner

Reply:
I really don't love it. IIRC in some situations we copied the `.d.ts` file on the build command to `/lib`. @cprussin anyway we can get around this?

Reply:
I'm going to pull this branch today and see if I can suggest anything else.  Subpath exports is generally how I've done this kind of thing in the past rather than re-exporting, but @cctdaniel mentioned he tried the subpath export and ran into issues.